src/RegExLib.Web/Endpoints/Expressions/Create.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Ardalis.ApiEndpoints;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Threading.Tasks;
+using RegExLib.SharedKernel.Interfaces;
+using RegExLib.Web.ApiModels;
+
+namespace RegExLib.Web.Endpoints.Expressions
+{
+  public class Create : BaseAsyncEndpoint<CreateExpressionCommand, CreateExpressionResult>
+  {
+    private readonly IRepository _repository;
+
+    public Create(IRepository repository)
+    {
+      _repository = repository;
+    }
+
+    [HttpPost("/api/expressions")]
+    [SwaggerOperation(
+      Summary = "Creates a new Expression",
+      Description = "Creates a new Expression",
+      OperationId = "Expression.Create",
+      Tags = new[] { "ExpressionEndpoint" })
+    ]
+    public override async Task<ActionResult<CreateExpressionResult>> HandleAsync([FromBody] CreateExpressionCommand request)
+    {
+      var expression = ExpressionDTO.ToExpression(request.ExpressionDto);
```

Comment:
I'm not sure there's a lot of utility provided by this static method call over just `new`ing up an `Expression` inline here. Later, if this gets too cumbersome (a la "Pain-Driven Development" per Steve), you could swap in AutoMapper.

Reply:
Done in commit 36854